cmd/kubelet/app/options/options.go

```diff
@@ -320,8 +320,6 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.MarkDeprecated("register-schedulable", "will be removed in a future version")
 	fs.StringVar(&f.ExperimentalMounterPath, "experimental-mounter-path", f.ExperimentalMounterPath, "[Experimental] Path of mounter binary. Leave empty to use the default mount.")
 	fs.MarkDeprecated("experimental-mounter-path", "will be removed in 1.25 or later. in favor of using CSI.")
-	fs.StringVar(&f.CloudConfigFile, "cloud-config", f.CloudConfigFile, "The path to the cloud provider configuration file. Empty string for no configuration file.")
```

Comment:
removing flags cause new kubelets that had they configured to fail, so we need to warn about this on the release notes

Reply:
@aojea do you mean that I should add `ACTION REQUIRED` to the first line of the current release note text?

Reply:
yep, I don't know what is the exact term but something like that